modules/baseline/variables.tf

```diff
@@ -1,33 +1,22 @@
-variable "central_budget_notification" {
-  type        = string
-  default     = ""
-  description = "Central or Shared email address for Budget Notifications"
+variable "aws_account_id" {
+  type = string
 }
-
-variable "config" {
-  type        = map(any)
-  description = "Map of configuration items"
+variable "allowed_regions" {
+  type = list(string)
 }
 
-variable "profile" {
-  type        = string
-  description = "AWS Shared Credentials Profile"
+variable "budget_limit_amount" {
+  type = number
 }
-
-variable "project" {
+variable "central_budget_notification" {
```

Comment:
```suggestion

variable "central_budget_notification" {
```

Reply:
This doesn't show as outdated for some reason, but whitespace has been added.